packages/features/ee/teams/components/v2/CreateNewTeam.tsx

```diff
@@ -1,112 +1,128 @@
+            defaultValue=""
+            rules={{
+              required: t("must_enter_team_name"),
+            }}
+            render={({ field: { value } }) => (
+              <>
+                <TextField
+                  className="mt-2"
+                  name="name"
+                  label={t("team_name")}
+                  value={value}
+                  onChange={(e) => {
+                    newTeamFormMethods.setValue("name", e?.target.value);
+                    if (newTeamFormMethods.formState.touchedFields["slug"] === undefined) {
+                      newTeamFormMethods.setValue("slug", slugify(e?.target.value));
+                    }
                   }}
-                  imageSrc={value}
+                  autoComplete="off"
                 />
+              </>
+            )}
+          />
+        </div>
+
+        <div className="mb-8">
+          <Controller
+            name="slug"
+            control={newTeamFormMethods.control}
+            rules={{ required: t("team_url_required"), validate: async () => await validateTeamSlug() }}
+            render={({ field: { value } }) => (
+              <TextField
+                className="mt-2"
+                name="slug"
+                label={t("team_url")}
+                addOnLeading={`${WEBAPP_URL}/team/`}
+                value={value}
+                onChange={(e) => {
+                  newTeamFormMethods.setValue("slug", slugify(e?.target.value), {
+                    shouldTouch: true,
+                  });
+                }}
+              />
+            )}
+          />
+        </div>
+
+        <div className="mb-8">
+          <Controller
+            control={newTeamFormMethods.control}
+            name="logo"
+            render={({ field: { value } }) => (
+              <div className="flex items-center">
+                <Avatar alt="" imageSrc={value || null} gravatarFallbackMd5="newTeam" size="lg" />
+                <div className="ml-4">
+                  <ImageUploader
+                    target="avatar"
+                    id="avatar-upload"
+                    buttonMsg={t("update")}
+                    handleAvatarChange={(newAvatar: string) => {
+                      newTeamFormMethods.setValue("logo", newAvatar);
+                    }}
+                    imageSrc={value}
+                  />
+                </div>
               </div>
-            </div>
-          )}
-        />
-      </div>
-      <div className="flex space-x-2">
-        <Button color="secondary" href="/settings" className="w-full justify-center">
-          {t("cancel")}
-        </Button>
-        <Button color="primary" type="submit" EndIcon={Icon.FiArrowRight} className="w-full justify-center">
-          {t("continue")}
-        </Button>
-      </div>
+            )}
+          />
+        </div>
 
-      {createTeamMutation.isError && <p className="mt-4 text-red-700">{createTeamMutation.error.message}</p>}
```

Comment:
I don't see any error handling for this added back in. I believe right now we don't show any error when the mutation fails, is that correct?

Reply:
re-added